baseframe/templates/baseframe/ajaxform.html

```diff
@@ -1,4 +1,8 @@
-{% from "baseframe/forms.html" import renderform, ajaxform, widget_ext_scripts, widgetscripts %}
+{% if theme == "material" %}
+	{% from "baseframe/forms-material.html" import renderform, ajaxform, widget_ext_scripts, widgetscripts %}
+{%- else %}
+    {% from "baseframe/forms.html" import renderform, ajaxform, widget_ext_scripts, widgetscripts %}
```

Comment:
I'd like to have a clean structure to these. Maybe `baseframe/mui/forms.html` and `baseframe/bootstrap3/forms.html`. There is also #104 that requires these templates to have `.html.jinja2` extensions. We might as well do that now, so make this `baseframe/mui/forms.html.jinja2`.

[Since a lot of other repositories refer to `baseframe/forms.html`](https://github.com/search?q=org%3Ahasgeek+from+%22baseframe%2Fforms.html%22&type=Code) we can help transition them by retaining a `forms.html` that simply imports everything from `baseframe/bootstrap3/forms.html`.

Reply:
Moved bootstrap3 and mui templates to separate folders.
Changed templates extensions to to .html.jinja2